arrow-benchmarks-fx/build.gradle

```diff
@@ -100,3 +100,7 @@ compareBenchmarksCI {
 repositories {
     maven { url "https://jitpack.io" }
 }
+
+compileKotlin {
+    kotlinOptions.jvmTarget = "1.8"
+}
```

Comment:
Thanks @nomisRev !! It shouldn't be necessary to include this conf because it should come from https://github.com/arrow-kt/arrow/blob/master/gradle/subproject.gradle that is applied via `apply from: "$SUB_PROJECT"` Please, could you try it without it?

Reply:
Yes, I think this is fine. It keeps showing up in my IDEA as an error, and sometimes will refuse to build from IDEA while it'll build fine from CLI.